src/options.rs

```diff
@@ -208,10 +192,65 @@ impl Orientation {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum ColorInversion {
+    Normal,
+    Inverted,
+}
+
+impl Default for ColorInversion {
+    fn default() -> Self {
+        Self::Normal
+    }
+}
+
+impl From<bool> for ColorInversion {
+    fn from(val: bool) -> Self {
+        match val {
+            true => Self::Inverted,
+            false => Self::Normal,
+        }
+    }
+}
+
```

Comment:
This could be removed if `Builder::with_invert_colors` would use the enum instead of a `bool`.

Reply:
fixed